code/game/turfs/simulated/wall/mineral_walls.dm

```diff
@@ -192,6 +192,11 @@
 	icon_state = "map-overspace"
 	fixed_underlay = list("space"=1)
 
+/turf/closed/wall/mineral/titanium/ship
+	icon = 'icons/turf/shuttle.dmi'
+	icon_state = "wall3"
```

Comment:
what the fuck are you doing with this subtype? I'm pretty sure that's the syndie shuttle wall icon, and if so, just use the actual plastitanium instead of starting a new era of retarded as var edits and subtypes